spec/cdn_source_spec.rb

```diff
@@ -190,6 +194,17 @@ def cleanup
         spec.name.should == 'BeaconKit'
         spec.version.should.to_s == '1.0.5'
       end
+
+      it 'does not attempt to access a version no in the version index' do
```

Comment:
nit: "not"

Reply:
Ah, I thought you meant replace `doesn\'t`.